src/command/setup.rs

```diff
@@ -61,7 +61,7 @@ mod os {
             .iter()
             .chain(&env_profile.as_ref().map(String::as_str))
             .fold(false, |prev, path| {
-                let profile = user_home_dir.join(path);
+                let profile = default_home_dir.join(path);
```

Comment:
This should be reverted as well, since the variable is named `user_home_dir` on line 54

Reply:
👍